llvm-bazel/zlib.bzl

```diff
@@ -36,7 +36,7 @@ llvm_zlib_external = repository_rule(
             default = Label("//deps_impl:zlib_external.BUILD"),
             allow_single_file = True,
         ),
-        "external_zlib": attr.label(
+        "external_zlib": attr.string(
```

Comment:
I feel like Bazel is doing the wrong thing here when converting the label to a string. It shouldn't be dropping the leading "@" if that is load-bearing

Reply:
Right. Agreed on this, maybe I'll take a look at the Label implementation and see what's going on. Either there's a bug or you're supposed to use the Label.relative() method somehow. But after a quick look at docs I couldn't figure it out and thought string would be good enough.

Reply:
Yeah I did the same and couldn't figure out what the "right" way. Can you file an issue/start a discussion?

Reply:
The function that does what you want appears to be here:

https://github.com/bazelbuild/bazel/blob/227da61a0a522e5c3e3cd3f05e2e2d3a0a0c1b00/src/main/java/com/google/devtools/build/lib/cmdline/Label.java#L438-L444

But this isn't exposed through the starlark bindings. I'll file a question with bazel to ask what the right solution is.

Reply:
Filed bazelbuild/bazel#13731.